index.html

```diff
@@ -1353,7 +1353,48 @@ <h4>Cross-site covert channel</h4>
       a video conferencing session will typically exert sustained pressure on the CPU that makes it harder to
       manipulate the pressure state in a predictive manner.
     </div>
-
+    <h4>Targeted deanonymization attacks</h4>
+    <p>
+      Targeted deanonymization attacks constitute a critical class of threats that jeopardize a users anonymity.
+      These attacks allow a malicious or partially compromised website (referred to as the “malicious site”) to
+      ascertain whether a website visitor possesses a specific public identifier, such as an email address or a
+      social media handle.
+    </p>
+    <p>
+      While anonymity may be a luxury for some, for certain individuals, it is far more than that—it is a matter
+      of survival. Consider for instance those who engage in political protests, work as journalists covering
```

Comment:
nit
"Consider for instance those who engage in political protests, work as journalists covering sensitive topics, etc"
maybe:
"Consider for instance those who engage with freedom of information."

Reply:
I think that is a bit weak and harder to understand

Reply:
it s just more generic